### DISCUSSION TITLE
Dịch hold-out (set)

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#74] [Terminology] [answered]
Từ này hay gặp trong phần kiểm định chéo. Mọi người có góp ý gì cách dịch không?
@mlbvn/handson-ml2

**Comment by rootonchair** [comment, marked as answer]
Lúc trước em thấy có dịch là `tập giữ lại`

**Reply by ngcthuong** [reply]
có vẻ ok.
